bundles/AdminBundle/Resources/public/js/pimcore/object/tags/manyToManyRelation.js

```diff
@@ -669,21 +670,35 @@ pimcore.object.tags.manyToManyRelation = Class.create(pimcore.object.tags.abstra
         return this.dataChanged;
     },
 
-    requestNicePathData: function (targets) {
+    requestNicePathData: function(targets, isInitialLoad) {
         if (!this.object) {
             return;
         }
+
+        var context = this.getContext();
+        var loadEditModeData = false;
+        if(isInitialLoad && this.fieldConfig.optimizedAdminLoading && context['containerType'] == 'object') {
```

Comment:
why did you add that here? optimizedAdminLoading is never available for manyToManyRelations

Reply:
good catch...I'll remove it ;)